manual_test_functions.py

```diff
@@ -40,6 +41,17 @@ async def main() -> None:
     for i, message in enumerate(messages):
         print(f"[{i}] {_message_summary(message)}")
 
+    # ATTENDANCE EXPORT
+
+    print("\nGetting a random attendance report...")
```

Comment:
Nitpicking, but it's for the first event returned.

Reply:
True, i adjusted the comment